openshiksha/settings.py

```diff
@@ -223,6 +223,7 @@
     'django.contrib.admindocs',
     'debug_toolbar',
     'django_extensions',
+    'corsheaders',
```

Comment:
With the deployment pattern I'm setting up, we won't need cors overrides in production, only when running locally, so you should only add this to installed apps when OPENSHIKSHA_ENV is local